src/main/webapp/WEB-INF/jsp/contributions/donate.jsp

```diff
@@ -247,4 +247,26 @@
             </script>
         </div>
     </div>
+
+    <!-- Get donations list using Covalent API -->
+    <script type="text/javascript">
```

Comment:
The `type` attribute should be [omitted](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-type) in HTML5:

```suggestion
    <script>
```